backend/src/game/dice_handler.py

```diff
@@ -0,0 +1,24 @@
+import backend.src.game.dice as dice
+import backend.src.game.constants as constants
+
+
+class DiceHandler(object):
+    dice_values = list()
+    __re_rolls_left = constants.DEFAULT_RE_ROLL_COUNT
```

Comment:
There's no reason here to use underscores. You're not creating a object-level variable that couldn't possibly be touched from the outside. The double underscore (which is different from dunder) also implies that it's a class level variable that could change at any given moment so it shouldn't be used outside of the internals for this object.